cmd/otelcol/config/collector/ecs_ec2_config.yaml

```diff
@@ -125,12 +125,12 @@ exporters:
     access_token: "${SPLUNK_ACCESS_TOKEN}"
     realm: "${SPLUNK_REALM}"
     correlation:
-#  # Logs
-#  splunk_hec:
-#    token: "${SPLUNK_HEC_TOKEN}"
-#    endpoint: "${SPLUNK_HEC_URL}"
-#    source: "otel"
-#    sourcetype: "otel"
+  # Logs
```

Comment:
```suggestion
  # Logs + Profiling
```

Reply:
Should we add this comment above the pipelines configuration instead instead of just the exporter?

Reply:
Should we add this comment above the pipelines configuration instead of just the exporter?

Reply:
It's just for consistency with comments for other exporters. We can move all of them to pipelines as well, maybe as follow up PR?

Reply:
Sounds good